src/items/deleteItemDialog.component.tsx

```diff
@@ -72,13 +75,42 @@ const DeleteItemDialog = (props: DeleteItemDialogProps) => {
         `Please move item to a system with Type: ${allowedSystemTypes.join(', ')} before trying to delete.`
       );
     }
-  }, [selectedRules, deleteItem, deletionRules, item, onChangeItem, onClose]);
+  }, [
+    selectedRules,
+    deleteItem,
+    deletionRules,
+    isPrivilegedUser,
+    item,
+    onChangeItem,
+    onClose,
+  ]);
 
   return (
     <Dialog open={open} maxWidth="lg">
       <DialogTitle sx={{ display: 'inline-flex', alignItems: 'center' }}>
         <WarningIcon sx={{ marginRight: 1 }} />
-        Delete Item
+        Delete Item{isPrivilegedUser ? ' as admin' : ''}
+        {isPrivilegedUser && (
+          <Tooltip
+            title={
+              'As an admin, you can bypass rules that prevent other users from deleting an item'
+            }
+            disableHoverListener={false}
+            aria-label={'admin-status-tooltip'}
```

Comment:
```suggestion
            data-testid={'admin-status-tooltip'}
```
Is this just used for tests? I don't think this is a good aria label for accessibility. Same for any others.